open_prices/proofs/models/price_tag.py

```diff
@@ -0,0 +1,150 @@
+from django.contrib.postgres.fields import ArrayField
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils import timezone
+
+from open_prices.common import constants, utils
+from open_prices.prices.models import Price
+from open_prices.proofs import constants as proof_constants
+
+from .proof import Proof
+
+
+class PriceTag(models.Model):
+    """A single price tag in a proof."""
+
+    UPDATE_FIELDS = ["bounding_box", "status", "price_id"]
+    CREATE_FIELDS = UPDATE_FIELDS + ["proof_id"]
+
+    proof = models.ForeignKey(
+        Proof,
+        on_delete=models.CASCADE,
+        related_name="price_tags",
+        verbose_name="The proof this price tag belongs to",
+    )
+    price = models.ForeignKey(
+        Price,
+        on_delete=models.SET_NULL,
+        related_name="price_tags",
+        null=True,
+        blank=True,
+        verbose_name="The price linked to this tag",
+    )
+    created = models.DateTimeField(
+        default=timezone.now, verbose_name="When the tag was created in DB"
+    )
+    updated = models.DateTimeField(
+        auto_now=True, verbose_name="When the tag was last updated"
+    )
+    bounding_box = ArrayField(
+        base_field=models.FloatField(),
+        verbose_name="Coordinates of the bounding box, in the format [y_min, x_min, y_max, x_max]",
+    )
+    status = models.IntegerField(
+        choices=constants.PRICE_TAG_STATUS_CHOICES,
+        null=True,
+        blank=True,
+        verbose_name="The annotation status. Possible values are: "
```

Comment:
this info should be in the choices enum. the verbose_name should just be a short label :)